app/scripts/services/util.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+angular.module('confRegistrationWebApp')
+    .factory('util', function util() {
+        var service = {
+            isUndefinedOrNull: function (obj) {
+                return angular.isUndefined(obj) || obj === null || obj === 'null';
+            },
+            isNumber: function (val) {
```

Comment:
@OzzieOrca Used _.isNaN instead of _isNumber, as _isNumber returns false, if the number is enclosed in quotes.